src/components/tables/DataTable.js

```diff
@@ -98,9 +110,12 @@ export default {
       const page = this.rowsPerPage === Object(this.rowsPerPage)
         ? this.rowsPerPage.value
         : this.rowsPerPage
-      return page === -1 ? this.value.length : this.page * page
+      return page === -1 ? this.length : this.page * page
     },
     filteredItems () {
+      if (this.totalItems)
```

Comment:
I can't put this on one line, if you mean `this.totalItems && return this.value`. That results in unexpected token for the `return`